content/en/docs/concepts/architecture/controller.md

```diff
@@ -154,8 +154,8 @@ controller does.
 
 ## {{% heading "whatsnext" %}}
 
-* Read about the [Kubernetes control plane](/docs/concepts/#kubernetes-control-plane)
-* Discover some of the basic [Kubernetes objects](/docs/concepts/#kubernetes-objects)
+* Read about the [Kubernetes control plane](/docs/reference/glossary/?all=true#term-control-plane)
```

Comment:
Does it make sense to link to a page here, not to the glossary. See the control plane link mentioned above.

Reply:
Link to Kubernetes objects:
https://kubernetes.io/docs/concepts/overview/working-with-objects/

Reply:
I think I'd link to https://kubernetes.io/docs/concepts/overview/components/#control-plane-components

I wonder if it makes sense to add a _new_ page https://kubernetes.io/docs/concepts/architecture/control-plane/ - if so, let's track that as a feature request.

Reply:
> I think I'd link to https://kubernetes.io/docs/concepts/overview/components/#control-plane-components
> 
> 
> 
> I wonder if it makes sense to add a _new_ page https://kubernetes.io/docs/concepts/architecture/control-plane/ - if so, let's track that as a feature request.

Agree with you. I had the same idea when I was doing this fixing.

Reply:
> Does it make sense to link to a page here, not to the glossary. See the control plane link mentioned above.

Perhaps a new control plane page like @sftim mentioned would be a better choice?

Reply:
I suggest to fix the links now and create a new issue/feature request for creating the page.

Reply:
Keep the current change until we have a new page such as  https://kubernetes.io/docs/concepts/architecture/control-plane/